Makefile.am

```diff
@@ -415,6 +415,14 @@ uninstall-local:
 	-rm $(DESTDIR)$(udevrulesdir)/$(udevrulesprefix)tpm-udev.rules
 endif
 
+install-exec-hook:
+	cd $(DESTDIR)$(libdir) && \
+		$(LN_S) libtss2-tcti-device.so libtss2-tcti-default.so
```

Comment:
It would be desirable to use `$(LN_S) -f`, see https://github.com/tpm2-software/tpm2-tss-engine/pull/63:
> Additionally I noticed that you cannot `make install` twice to the same directory because creating the symlink [...] a second time fails. Adding `--force` overwrites the existing symlink (exactly like the standard Makefile installation procedure overwrites existing files), making updating an existing installation possible.

This also concerns installation of the [`Tss2_TctiLdr_Initialize_Ex` man page in l. 406](https://github.com/tpm2-software/tpm2-tss/blob/8506c7d155ac16497dd378b61227223767b6cdc9/Makefile.am#L406).